### PR TITLE
fix: set rename error message below the text field

### PR DESCRIPTION
**Pull request by nborde-CSM**

### CODE REVIEW

File: src/inputs/EditableLabel/EditableLabel.js [hunk 1]
```diff
@@ -22,18 +29,15 @@ const useStyles = makeStyles((theme) => ({
     overflow: 'hidden',
     textOverflow: 'ellipsis',
   },
-  textField: {
+  OutlinedInput: {
```
Comment (csm-thu) [inline src/inputs/EditableLabel/EditableLabel.js:32]: To be replaced by `outlinedInput` to stay consistent with other classnames :wink:

File: src/inputs/EditableLabel/EditableLabel.js [hunk 2]
```diff
@@ -108,33 +112,38 @@ export const EditableLabel = (props) => {
...
+            }}
+            onKeyDown={(event) => {
+              if (event.key === 'Escape') {
+                setEditing(false);
+              }
+            }}
+            onFocus={(event) => {
+              event.stopPropagation();
+            }}
+            {...textFieldProps}
```
Comment (csm-thu) [inline src/inputs/EditableLabel/EditableLabel.js:141]: The name `textFieldProps` is no longer relevant if the TextField has been removed (and replaced here by an `OutlinedInput`. Do we still need additional props for this element ?

File: src/inputs/EditableLabel/EditableLabel.js [hunk 3]
```diff
@@ -108,33 +112,38 @@ export const EditableLabel = (props) => {
...
+          />
+          <FormHelperText error id="my-helper-text">
```
Comment (csm-thu) [inline src/inputs/EditableLabel/EditableLabel.js:143]: Please change `my-helper-text` (or remove the id if it is not useful) :smile:

File: src/inputs/EditableLabel/EditableLabel.js [hunk 4]
```diff
@@ -108,33 +112,35 @@ export const EditableLabel = (props) => {
...
-          onChange={checkNewValueOnChange}
-          onBlur={(event) => {
-            stopEdition(event);
-          }}
-          onClick={(event) => event.stopPropagation()}
-          onKeyPress={(event) => {
-            if (event.key === 'Enter') {
+        <FormControl>
+          <OutlinedInput
+            id="component-outlined"
```
Comment (esasova) [inline src/inputs/EditableLabel/EditableLabel.js:117]: Is the id used elsewhere ? :slightly_smiling_face:

File: src/inputs/EditableLabel/EditableLabel.js [hunk 5]
```diff
@@ -50,7 +54,7 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 export const EditableLabel = (props) => {
-  const { value, onNewValue, checkValue, labels, typographyProps, textFieldProps } = props;
+  const { value, onNewValue, checkValue, labels, typographyProps } = props;
```
Comment (csm-thu) [inline src/inputs/EditableLabel/EditableLabel.js:57]: `textFieldProps` has not been removed at the bottom of the file, in `EditableLabel.propTypes`

**Pull request merged**